### ISSUE TITLE
In the single-answer workflow, I cannot change my answer after selecting any of the other options.

### ISSUE THREAD

**Issue by chelseatroy** [zooniverse/mobile#298]
In order to fix an issue, we need to be able to reproduce it. To give us the best chance of reproducing your issue, please follow these steps:

Please check that you currently have the latest mobile app version (2.6.1 as of July 2, 2019) installed. If not, please update.
In your issue, please include:

1. The version of Android or iOS that you are testing on: iOS 13
1. The device that you are testing on (for example, iPhone 7 or Samsung Galaxy s9): iPhone X Simulator
1. What you did.

I went to a single-answer workflow in the app and selected an answer for the first subject. I then wanted to _change_ my answer, but I could not; I could only submit.

**Comment by camallen** [comment]
I just ran into this issue on Bash the Bug project's mobile enabled workflow. 

I'm using Latest version of the app, `2.6.8`
The version of Android or iOS that you are testing on: `Android 10 & 7.0`
The device that you are testing on (for example, iPhone 7 or Samsung Galaxy s9): `Moto G4 Plus & Pixel 3a`
What you did.
```
Visited Bash the Bug mobile project and started classifying, chose one of the answers, tried to change it to another but couldn't. Had to abort my classification or submit a known incorrect answer.
```

**Comment by chelseatroy** [comment]
Yep, this one's pretty high priority to fix.

C

Chelsea Troy
chelseatroy.com

**Comment by chelseatroy** [comment]
THis is fixed in this PR: https://github.com/zooniverse/mobile/pull/309/files

**Comment by lcjohnso** [comment]
Closed by #309
